protos/sigstore_trustroot.proto

```diff
@@ -117,3 +118,50 @@ message TrustedRoot {
         // A set of trusted timestamping authorities.
         repeated CertificateAuthority timestamp_authorities = 5;
 }
+
+
+// SigningConfig represents the trusted entities/state needed by Sigstore
+// signing. In particular, it primarily contains service URLs that a Sigstore
+// signer may need to connect to for the online aspects of signing.
+message SigningConfig {
```

Comment:
Flagging: which, if any, of these should be required? I think `ca_url` and `oidc_url` at minimum.

Reply:
I could see a non-keyless config existing? Do we care about those people? Starting everything optional seems safest? There's a bit of ambiguity of what a proper "sigstore" signature means -- cause cosign allows everything.

Reply:
Yeah, good point. The requirements are purely hints at the DSL level here anyways, so we can leave them out for now and expect clients to enforce their own requirements here.

Reply:
sgtm